grails-app/assets/javascripts/asinha/payer-create.js

```diff
@@ -94,8 +94,11 @@ function PayerCreate() {
         var url = document.querySelector("form").getAttribute("action");
 
         $.post(url, infosPayer, function(response) {
-            console.log(response);
-            window.location.href = document.querySelector("form").getAttribute("data-redirect");
+            if (response.success) {
```

Comment:
Utilizar earlyReturn e sempre tratamos o erro, o sucesso deve seguir o caminho normal do método
```suggestion
            if (!response.success) {
             Tratativa do Erro
             return
            }
            Tratativa do Sucesso
```